DockDoor/Views/Hover Window/WindowPreview.swift

```diff
@@ -57,45 +57,31 @@ struct WindowPreview: View {
 
     private func windowContent(isMinimized: Bool, isHidden: Bool, isSelected: Bool) -> some View {
         Group {
-            if isMinimized || isHidden {
-                let width = maxWindowDimension.x > 300 ? maxWindowDimension.x : 300
-                let labelText = isMinimized ? "Minimized" : "Hidden"
-
-                HStack(spacing: 16) {
-                    Image(systemName: "eye.slash.fill")
-                        .font(.system(size: 20))
-                        .foregroundColor(.secondary)
-
-                    Divider()
-
-                    VStack(alignment: .leading) {
-                        Text(labelText)
-                            .font(.caption)
-                            .foregroundStyle(.secondary)
-
-                        TheMarquee(width: width - 100, secsBeforeLooping: 2, speedPtsPerSec: 30, nonMovingAlignment: .leading) {
-                            Text(windowInfo.windowName ?? "\(labelText) window")
-                                .font(.system(size: 12, weight: .medium))
-                                .foregroundStyle(.primary)
+            if let cgImage = windowInfo.image {
+                Image(decorative: cgImage, scale: 1.0)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .overlay(
+                        Group {
+                            if isMinimized || isHidden {
+                                fluidGradient().opacity(0.225)
+                                HStack {
+                                    Image(systemName: "eye.slash.fill")
+                                    Text(isMinimized ? "Minimized" : "Hidden")
+                                }
+                                .font(.caption)
+                                .italic()
+                                .foregroundStyle(.secondary)
+                            }
```

Comment:
@hasansultan92 what about that?

Reply:
So this is more on the UI side of things, I was not aware we were having this issue to begin with, and something @ejbills has been contributing to mostly. I don't have an issue with this UI look but I do believe we should add more opacity to make the minimized window preview almost opaque

Reply:
I don't understand why a preview of a minimized window should be blurry. In my opinion the main use of DockDoor is for minimized windows, normal windows are usually already visible

Reply:
hmm, my thought would be the opposite. If I have a minimized window, that's because I don't want others to see or myself that it exists and I most certainly would not want to see the preview either, defeats the purpose of minimizing a window, though I also see your point. 

Perhaps we can introduce a state of some sort that can allow what you want and perhaps what I want? If the state is flipped we can have your preview, and if the state is true, we can have the preview I mentioned. Of course this can be vice versa too.

Reply:
hmm, I see your point @ShlomoCode. Thoughts on this @ejbills?

Reply:
> I don't like this overlay ... It seems quite clunky

Don't get how it was clunky, but the UI was not final. I pushed a changed version which I am happy with, and will be the final UI for minimized/hidden windows. Open to constructive criticism of course.

@ShlomoCode this PR will be merged tomorrow pending some testing.